orkaudio/filters/LiveStream/json.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#ifndef JSON_HEADER_FILE_H
+#define JSON_HEADER_FILE_H
+
+/**
+* Need to undef snprintf other MSVC throws error that it is not part of std::
+* Somewhere when including pjsua2.hpp this gets set.
+*/
+#undef snprintf
+#include <nlohmann/json.hpp>
+
+#endif
```

Comment:
You can skip this file completely. Just include `#include <nlohmann/json.hpp>` directly. This isn't a MSVC app, hence you can skip the logic of `snprintf`.

Reply:
removed